src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/EmailPreview/EmailPreview.tsx

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import type { FC } from 'react'
+import styled from 'styled-components'
+import { themeSpacing } from '@amsterdam/asc-ui'
+import FormFooter from 'components/FormFooter'
+import ModalHeader from '../ModalHeader/ModalHeader'
+
+const ModalContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const StyledFormFooter = styled(FormFooter)`
+  height: auto;
+  div {
+    padding-left: ${themeSpacing(4)};
+    div {
+      padding-left: 0;
+    }
+  }
```

Comment:
This construct is very unreliable. If the internal structure of the `FormFooter` component changes, these rules might lead to a broken GUI. Isn't there a way to target elements by class name or have components that can be targeted as styled components?

Reply:
I will adjust the FormFooter component by adding a class then